html/main.js

```diff
@@ -0,0 +1,129 @@
+const cardColors = [
+    "red",
+    "orange",
+    "yellow",
+    "olive",
+    "green",
+    "teal",
+    "blue",
+    "violet",
+    "purple",
+    "pink",
+    "brown",
+    "grey",
+    "black"
+];
+
+$(".ui.card").hover(
+    function () {
+        $(this).addClass("raised");
+    }, function () {
+        $(this).removeClass("raised");
+    }
+);
+
+$(".ui.transparent.fluid.input").focusin(
+    function () {
+        $(this).children("div").show(100);
+    }
+).focusout(
+    function () {
+        $(this).children("div").hide(100);
+    }
+);
+
+$(".sortable").sortable({
+    placeholder: "card"
+});
+
+$(".add-command").click(function () {
+    $(".ui.modal.add-command-modal").modal("show");
+});
+
+$("#cards-parent").on("click", ".delete", function () {
+    $(this).parents(".ui.card").remove();;
+});
+
+$(".ui.modal.add-command-modal").modal("setting", "transition", "horizontal flip").modal({
+    closable: true,
+    onApprove() {
+        // If title empty, return false
+        let parentContent = $(this).children(".content");
+        let inputHeader = parentContent.find("input[name ='header']");
+        if (inputHeader.val() === "")
+        {
+            inputHeader.focus();
+            return false;
+        }
+
+        // Else call method below
+        submitModalNewCommand(parentContent);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['submitModalNewCommand' was used before it was defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)